src/dbus_api/util.rs

```diff
@@ -123,6 +123,7 @@ where
     prop_changed
         .changed_properties
         .insert(prop_name.into(), Variant(Box::new(new_value)));
+    prop_changed.interface_name = STRATIS_BASE_SERVICE.into();
```

Comment:
Maybe this is supposed to be the interface specific to the property instead? e.g., ```org.storage.stratis1.filesystem```, ```org.storage.stratis1.pool```. I didn't see much information in the source code about this, but that seems more plausible to me, as we know that objects can support multiple interfaces and it is possible that an identical property name might occur in each interface, so the particular interface would be the best way to disambiguate among the two properties.

Reply:
fixed